### PR TITLE
Addon cleanup

### PR DESCRIPTION
**Pull request by seebeen**
Implemented changes suggested by @stasadev in #38

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,22 +15,72 @@ It's based on [MinIO official image](https://hub.docker.com/r/minio/minio) and [
...
+
+Since MinIO is S3 compatible you can use [AWS PHP SDK](https://packagist.org/packages/aws/aws-sdk-php). Install it with composer:
 
 ```bash
 $ ddev composer require aws/aws-sdk-php
 ```
-Project docker instances can access MinIO api via `http://minio:10101`
 
-### Public access
+### Basic usage
```
Comment (rfay) [inline README.md:45]: This is great stuff. If there's not already a test that does exactly this, please consider adding one.

**Review by rfay — APPROVED** [review]
Looking great to me, thanks!

Don't forget the cleanup changing URLs and `ddev get` to ddev/ddev-minio all through the README, etc.

**Review by stasadev — APPROVED** [review]
Great stuff!

Yeah, I also wanted to mention, we need this in the README.md:

````
## Installation

```bash
$ ddev get ddev/ddev-minio
$ ddev restart
```
````

**Comment by seebeen** [comment]
> Great stuff!
> 
> Yeah, I also wanted to mention, we need this in the README.md:
> 
> ```
> ## Installation
> 
> ```bash
> $ ddev get ddev/ddev-minio
> $ ddev restart
> ```
> ```

Done :)

**Review by stasadev — APPROVED** [review]
Well, then let's merge it.

**Pull request merged**